### PR TITLE
[0.3.0] - 2019-02-11

### PR DESCRIPTION
**Pull request by avoloshko**
### Added
* ability to store unique name when creating/editing verifier
* option to enable and disable verifier (`active` flag): 
  this will affect only voting process, disabled verifier can still withdraw etc.

### Changed
* checking if verifier is created is now done by checking if `id` is not empty

### Removed
* `created` field was removed from verifier structure

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
...
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.3.0] - 2019-02-11
+### Added
+* ability to store unique name when creating/editing verifier
+* option to enable and disable verifier (`active` flag): 
+  this will affect only voting process, disabled verifier can still withdraw etc.
+
+### Changed
+* checking if verifier is created is now done by checking if `id` is not empty
```
Comment (enddynayn) [inline CHANGELOG.md]: `*` ----> `-`

**Comment by DZariusz** [comment]
👍

**Comment by therevoltingx** [comment]
What's up with this PR @enddynayn @avoloshko ?

**Comment by enddynayn** [comment]
👍

**Pull request merged**